lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -450,6 +456,23 @@ def _populate_from_source(self, source_data, using_current_cache):
                     cacheable_results[vm_guest_id]['hosts'].append(current_host)
                     self.inventory.add_child(vm_guest_id, current_host)
 
+                    # Based on folder of virtual machine
```

Comment:
Please see this - https://github.com/ansible/ansible/pull/52782

Reply:
Oh, sorry, that should not have been there. Didn't notice, that I accidentally committed that change too...